modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -661,10 +661,11 @@ public Assignment putAssignment(User.ID userID, Application.Name applicationName
             //before inserting the new data, we have to manually delete a corresponding row
```

Comment:
Perhaps we should remove complete trace of user_bucket_index code... Similar to other places/changes...